From 8bf90c1469c5e3fcb51463dec56a030ff269a35b Mon Sep 17 00:00:00 2001
From: Stefan Metzmacher <metze@samba.org>
Date: Wed, 22 Jun 2016 20:38:01 +0200
Subject: [PATCH 1100/1104] dcerpc.idl: remove unused
 DCERPC_NCACN_PAYLOAD_MAX_SIZE

BUG: https://bugzilla.samba.org/show_bug.cgi?id=11948

Signed-off-by: Stefan Metzmacher <metze@samba.org>
Reviewed-by: Ralph Boehme <slow@samba.org>
(cherry picked from commit d9e242e9035c15e49b041afc61e5a4a08877f289)
(cherry picked from commit a9a1ec7ad4312f3dea93ca9dbb6d9d67ad5c18ea)
---
 librpc/idl/dcerpc.idl | 1 -
 1 file changed, 1 deletion(-)

diff --git a/librpc/idl/dcerpc.idl b/librpc/idl/dcerpc.idl
index 5e0f919..527804d 100644
--- a/librpc/idl/dcerpc.idl
+++ b/librpc/idl/dcerpc.idl
@@ -535,7 +535,6 @@ interface dcerpc
 	const uint32 DCERPC_FRAG_MAX_SIZE  = 5840;
 	const uint8 DCERPC_AUTH_LEN_OFFSET = 10;
 	const uint8 DCERPC_NCACN_PAYLOAD_OFFSET = 16;
-	const uint32 DCERPC_NCACN_PAYLOAD_MAX_SIZE = 0x400000; /* 4 MByte */
 
 	/*
 	 * See [MS-RPCE] 3.3.3.5.4 Maximum Server Input Data Size
-- 
2.9.0

